docs/source/installation.rst

```diff
@@ -0,0 +1,146 @@
+Installation Guide
+************************
+
+Background Information
+========================
+The following section provides some background information on conda environments, git commands,
+and a package installation manager for python: PIP. If you've set these up already, you can skip directly to
+the *Installation* section
+
+
+Conda Environments
+------------------
+Conda environments are virtual environments used to separate projects and their respective
+package dependencies. These virtual environments contain distinct python interpreters, allowing packages
+installed within a given environment to be independent of the system interpreter.
+To install conda, go to the following website, and install the version
+needed for your operating system (OS).
+
+https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html#installing-conda-on-a-system-that-has-other-python-installations-or-packages
+
+General Commands
+~~~~~~~~~~~~~~~~
+To create conda environments, run the following
+
+``conda env create -n <name of environment> python=<desired python version>``
+
+For example, if I wanted to create an environment named ``mesmerize``, with python version 3.9,
+you can run:
+
+``conda env create -n mesmerize python=3.9``
+
+To see a list of existing conda environments, run the following:
+
+``conda env list``
+
+To activate a specific environment, run the following:
+
+``source activate <name of environment>``
+
+Git Commands
+----------------
+Git commands allow you to access, clone, modify, and update github repositories. To install
+Git on your computer, go to the following website, and install the version needed for your
+operating system (OS).
+
+https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
+**Note for Windows Users**
+
+Windows powershell gives me many issues when trying to install git within the powershell terminal,
+so I would recommend installing Git Bash Terminal instead, and running git commands from with Git Bash.
+
+To download, go to the following website and follow the directions
+
+https://www.educative.io/answers/how-to-install-git-bash-in-windows
+
+Package Installation Manager (PIP)
```

Comment:
mixing conda and pip packages in a conda env gets messy, so that's why I don't think it should belong here. They should learn about venvs elsewhere.

Reply:
Especially since we want users to eventually transition to using mesmerize-core in notebook, it might be better to keep consolidated sources of info on conda/pip somewhere. We can move this stuff to another section (at the bottom of the installation docs if you like), but I think users would have more success if they had one place to look for this stuff. Knowing what to google is still a struggle